ee/desktop/menu/menu.go

```diff
@@ -96,7 +99,7 @@ func (m *menu) getMenuData() *MenuData {
 // SetDefaults ensures we have the desired default values.
 func (md *MenuData) SetDefaults() {
 	if md.Icon == "" {
-		md.Icon = KolideDesktopIcon
+		md.Icon = TranslucentIcon
```

Comment:
I'd have expected `DefaultIcon` here (and maybe translucent in the initial)

Reply:
Sure, I wasn't exactly sure what was best choice here.